src/main/java/com/github/netelli/JDBCStuding.java

```diff
@@ -1,10 +1,13 @@
 package com.github.netelli;
 
 import com.github.netelli.model.Product;
+import org.apache.log4j.Logger;
 
 import java.util.List;
 
 public class JDBCStuding {
+    private final static Logger logger = Logger.getLogger(JDBCStuding.class);
+
     public static void main(String[] args) throws Exception {
         try (ProductsDAO productsDAO = new ProductsDAO("jdbc:h2:mem:test")) {
```

Comment:
add message about start of processing/work.

in the end of main() method add message about end of processing.